lib/rules/media-feature-name-no-vendor-prefix/index.mjs

```diff
@@ -30,25 +30,41 @@ const rule = (primary, _secondaryOptions, context) => {
 				return;
 			}
 
-			const matches = atRule.toString().match(/(?:min-|max-)?-[a-z-]+device-pixel-ratio/gi);
+			const features = [
```

Comment:
[suggest] How about extracting a constant for the features outside the loop like this?

```js
const FEATURES = [ /*...*/ ];

/** @type {import('stylelint').Rule} */
const rule = (primary) => {
  // ...

  const matches = atRuleString.match(new RegExp(`(?:${FEATURES.join('|')})`, 'gi'));
```